Monika After Story/game/script-topics.rpy

```diff
@@ -5523,7 +5523,19 @@ label monika_100k:
     return
 
 init 5 python:
-    addEvent(Event(persistent.event_database,eventlabel='monika_kizuna',prompt="Virtual Youtuber?", category=['misc'],random=False,unlocked=False,pool=True,action=EV_ACT_UNLOCK,conditional="seen_event('greeting_hai_domo')"))
+    addEvent(
+        Event(persistent.event_database,
+            eventlabel='monika_kizuna',
+            prompt="Virtual Youtuber?",
+            category=['misc'],
+            random=False,
+            unlocked=False,
+            pool=True,
+            action=EV_ACT_POOL,
+            conditional="seen_event('greeting_hai_domo')",
+            rules={"no unlock": None}
```

Comment:
Since the action is to `pool` it, `pool` should initially be False.

Should also drop the no unlock rule here.

In discord, what I meant was **either** have `pool=False` and change action to POOL, or keep `pool` and `action` as they were but add a no unlock rule. Dont do both.